src/main/webapp/WEB-INF/views/products/addProduct.jsp

```diff
@@ -0,0 +1,21 @@
+<head>
+    <title>Add product</title>
+</head>
+<body>
+<h1>Add product</h1>
+<h1>please add information</h1>
+<h2 style="color: blue">${message}</h2>
+<form method="post" action="${pageContext.request.contextPath}/products/addProduct">
+    Name: <input type="text" name="name">
+    Price: <input type="number" step="0.01" name="price">
```

Comment:
According to checklist we should make here a check whether the field empty and use jsp tag for it.

Reply:
Thx. Fixed.